### PR TITLE
Issue/89 restore config

### PR DESCRIPTION
**Pull request by mzxyz**
- support `dev`, `kauri` and `rimu` genesis config

closes #97

### CODE REVIEW

**Review by cowboy-bebug — COMMENTED** [review]
Do we want `genesis.json` configs instead of hardcoding them in scripts?

File: cli/src/chain_spec/mod.rs [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+				PLUG_ASSET_ID,
+				SYLO_ASSET_ID,
+				CERTI_ASSET_ID,
+				ARDA_ASSET_ID,
+				NEXT_ASSET_ID,
+			],
+			initial_balance: 10u128.pow(18 + 9), // 1 billion token with 18 decimals
+			endowed_accounts: endowed_accounts.clone(),
+			next_asset_id: NEXT_ASSET_ID,
+			// create_asset_stake: 1000, // removed upstream
```
Comment (jordy25519) [inline cli/src/chain_spec/mod.rs]: We can safely remove this

File: cli/src/chain_spec/dev.rs [hunk 2]
```diff
@@ -0,0 +1,43 @@
+use super::{config_genesis, get_account_id_from_seed, get_authority_keys_from_seed, ChainSpec, NetworkKeys};
+
+/// kauri genesis config
```
Comment (jordy25519) [inline cli/src/chain_spec/dev.rs]: `//! Dev genesis config`

File: cli/src/chain_spec/dev.rs [hunk 3]
```diff
@@ -0,0 +1,43 @@
+use super::{config_genesis, get_account_id_from_seed, get_authority_keys_from_seed, ChainSpec, NetworkKeys};
```
Comment (jordy25519) [inline cli/src/chain_spec/dev.rs:19]: Don't forget to add license header

File: cli/src/chain_spec/rimu.rs [hunk 4]
```diff
@@ -0,0 +1,102 @@
+use super::{config_genesis, ChainSpec, NetworkKeys};
+use hex_literal::hex;
+use primitives::crypto::UncheckedInto;
+
+/// rimu genesis config
```
Comment (jordy25519) [inline cli/src/chain_spec/rimu.rs]: For module comment should do this: `//! Rimu genesis config`

File: cli/src/lib.rs [hunk 5]
```diff
@@ -41,10 +41,10 @@ use transaction_factory::RuntimeAdapter;
 pub enum ChainSpec {
 	/// Whatever the current runtime is, with just Alice as an auth.
 	Development,
-	/// Whatever the current runtime is, with simple Alice/Bob auths.
-	LocalTestnet,
-	/// Whatever the current runtime is with the "global testnet" defaults.
-	StagingTestnet,
+	/// The CENNZnet Kauri testnet.
+	CennznetKauri,
+	/// The CENNZnet Rumi testnet.
```
Comment (jordy25519) [inline cli/src/lib.rs]: Rumi -> Rimu

**Review by jordy25519 — CHANGES_REQUESTED** [review]
Looks good, small things like some doc comments need fixing.

Please generate the json blobs for kauri and rimu. Into a `genesis/` folder or similar should do
e.g.
1) `./target/release/cennznet build-spec --chain=kauri > kauri.json`
2) `./target/release/cennznet build-spec --chain kauri.json --raw > kauri.raw.json`

(Useful link: https://substrate.dev/docs/en/tutorials/start-a-private-network-with-substrate#creating-the-chainspec)

We can fix the Rimu keys in a follow up PR

---

**Update**: Don't need to upload the readable json

**Review by jordy25519 — APPROVED** [review]
LGTM, needs fix for merge conflicts though

**Pull request merged**